app/assets/javascripts/application.js

```diff
@@ -32,11 +32,8 @@
 //= require external_scripts/jquery.fileupload.js
 //= require dataTables/jquery.dataTables
 //= require dataTables/bootstrap/2/jquery.dataTables.bootstrap
-//= require underscore
-//= require backbone
```

Comment:
So pumped we get to remove this finally!

Why do we need this file at all?

Reply:
Honestly I don't think we do! It just seemed like a big move to remove it. I'll dump it and see if anything breaks. Love to delete.